public/app/features/scenes/querying/SceneQueryRunner.ts

```diff
@@ -169,6 +172,53 @@ export class SceneQueryRunner extends SceneObjectBase<QueryRunnerState> {
   private onDataReceived = (data: PanelData) => {
     this.setState({ data });
   };
+
+  private applyPanelTimeOverrides(panel: { timeFrom?: string; timeShift?: string }, timeRange: TimeRange): TimeRange {
+    let newTimeRange = timeRange;
+
+    if (panel.timeFrom) {
+      const timeFromInterpolated = sceneGraph.interpolate(this, panel.timeFrom);
+      const timeFromInfo = rangeUtil.describeTextRange(timeFromInterpolated);
+      if (timeFromInfo.invalid) {
+        return newTimeRange;
+      }
+
+      if (_isString(timeRange.raw.from)) {
```

Comment:
Can we just use `typeof timeRange.raw.from === 'string'?

Reply:
sure! I reused the existing code, but we should be careful to not add unnecessary dependencies to make easier the scene packaging. Thanks for the suggestion! 🙏